dali/base/dadfs.cpp

```diff
@@ -2679,6 +2679,7 @@ class CDistributedFileBase : implements INTERFACE, public CInterface
     unsigned defaultTimeout;
     bool dirty;
     Owned<IRemoteConnection> superOwnerLock;
+    bool external = false;
```

Comment:
minor: moving this field before superOwnerLock would save 8 bytes of memory per object.